README.md

```diff
@@ -1,4 +1,293 @@
+            code: <cat-allocation-spec>
+            # L3 allocation spec for the data path when CDP is enabled (optional)
+            data: <cat-allocation-spec>
+        mbAllocation:
+          # MB allocation spec of the class
+          <cache-ids>: <mb-allocation-spec>
+```
+
+| Field | Format | Example | Description |
+| ----- | ------ | ------- | ----------- |
+| `<partition-name>` | string | `exclusive` | Name of a higher level RDT partition.
+| `<class-name>`     | string | `guaranteed` | Name of an RDT class, mapping to a directory in the resctrl fs.
+| `<cache-ids>`      | cpuset (string) | `0,2,4,8-11` | Set of cache ids. Special value 'all' denotes a default used for cache "all the reset".
+| `<cat-allocation-spec>` | percentage (string) | `"60%"` | Cache allocation spec, may be specified as relative (percentage) or absolute (bitmask). An absolute bitmask must be contiguous.
+                          | hex bitmask (string) | `"0xf0"` |
+                          | bit numbers (string) | `"0-3"` |
+| `<mb-allocation-spec>` | list of strings | `[50%, 1000MBps]` | Memory bandwidth allocation spec, separarate values for percentage and MBps based allocation. The *MBps* value is in effect when resctrl is mounted with `-o mba_MBps`.
+
+### Short forms
+
+The configuration accepts shortforms in order to allow easier and more readable
+configuration of the common and simple use cases.
+
+1. Separate unified/code/data specs can be omitted, when no separate CDP config
+   is desired, i.e.
+
+      ```
+          <cache-ids>: <cat-allocation-spec>
+
+      ```
+
+   is equal to
+
+      ```
+          <cache-ids>:
+            unified: <cat-allocation-spec>
+      ```
+
+1. `<cache-ids>` may be omitted if no cache id specific configuration (and no
+   CDP config for CAT) is desired, i.e.
+
+      ```
+        l3Allocation: "60%"
+        mbAllocation: ["50%"]
+      ```
+
+   is equal to
+
+      ```
+        l3Allocation:
+          all:
+            unified: "60%"
+        mbAllocation:
+          all: ["50%"]
+      ```
+
+### Examples
+
+Below is a config snippet that would allocate (ca.) 60% of the L3 cache lines
+exclusively to the guaranteed class. The remaining 40% L3 is for burstable and
+besteffort, Besteffort getting only 50% of this. guaranteed class gets full
+memory bandwidth whereas the other classes are throttled to 50%.
+
+```yaml
+options:
+  l2:
+    optional: true
+  l3:
+    optional: true
+  mb:
+    optional: true
+partitions:
+  exclusive:
+    # Allocate 80% of all L2 cache IDs to the "exclusive" partition
+    l2Allocation: "80%"
+    # Allocate 60% of all L3 cache IDs to the "exclusive" partition
+    l3Allocation: "60%"
+    mbAllocation: ["100%"]
+    classes:
+      guaranteed:
+        # Allocate all of the partitions cache lines and memory bandwidth to "guaranteed"
+        l2Allocation: "100%"
+        l3Allocation: "100%"
+        #mbAllocation: ["100%"]
```

Comment:
#mbAllocation is commented out in the example config. (The same in burstable and besteffort.) Maybe better removed or explained.

Reply:
Good point. Explained in a comment